public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Import font awesome icon library -->
+    <script src="https://kit.fontawesome.com/490bacaf0b.js" crossorigin="anonymous"></script>
```

Comment:
Glad to see someone else imports fontawesome through the HTML, looked at the react version and it looks too complicated 😆

Reply:
Tbh, I'd not even considered using react to import the icons.  Good to know the alternative doesn't look like it's worth doing!